src/app/Fake.Sql.FluentMigrator/Sql.FluentMigrator.fs

```diff
@@ -0,0 +1,316 @@
+﻿namespace Fake.Sql
+
+/// Contains functions to run FluentMigrator
```

Comment:
Still would love to see a small sample here in the xml-comments

Reply:
Yes, I have the sample done, I just need to make sure it's working properly, then I'll add it here.